docs/1.8.9/sbo.md

```diff
@@ -0,0 +1,138 @@
+* Inquis Line - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Burrow Line - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Guess Line - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Line Width - Irrelevant
+* Mythos HP - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Share Inquisitor - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Send Text On Inq Spawn - Irrelevant
+* Send Test Inq Message - ?
+* Highlight Inquis - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Inquis/Lootshare Color - Irrelevant
+* Inquis Lootshare Circle - Irrelevant
+* Inquis Lootshare Cylinder - Irrelevant
+* Add Warps - Irrelevant
+* Add Stonks Warp - Irrelevant
+* Chim Message - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Custom Chim Message Text - Irrelevant
+* Chim Message Test - Irrelevant
+* Reset Custom Chim Message - Irrelevant
+* Rare Drop Announcer - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Loot Screen Announcer - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Show Price Title - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+
+### Party Commands
+* Party Commands - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Warp Party - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Allinvite - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Party Transfer - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Promote/Demote - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Ask Carrot - ?
+* Time Check - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Check Tps - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Diana Party Commands - ?
+
+### Quality of Life
+* Effects For Blaze - ?
+* Parrot Level - Irrelevant
+* Formatted Bridge Bot - Irrelevant
+* Bridge Bot Name - Irrelevant
+* Copy Rare Drop - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Find Dragon Lair - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Jacob Message Hider - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Clean Diana Chat - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Hide AutoPet Messages - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Hide Sacks Message - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Clickable Messages To Invite a Player - ?
+* Pickup Log Overlay - [Biscuit & Fix3dll's SkyblockAddons Unofficial](https://modrinth.com/mod/skyblockaddons-unofficial) / [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Hide Tipped Players - ?
+* Carnival Redstone Lamp Helper - ?
```

Comment:
SkyHanni

Reply:
Where? I cannot seam to find it?
![image](https://github.com/user-attachments/assets/ed6f6a36-98c4-4d94-8886-4a501990597f)
![image](https://github.com/user-attachments/assets/aaba088a-fa5c-4ce9-942a-e2d5335ffbe6)

Reply:
> Where? I cannot seam to find it? ![image](https://private-user-images.githubusercontent.com/55346310/381803076-ed6f6a36-98c4-4d94-8886-4a501990597f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzOTMwOTMsIm5iZiI6MTczMDM5Mjc5MywicGF0aCI6Ii81NTM0NjMxMC8zODE4MDMwNzYtZWQ2ZjZhMzYtOThjNC00ZDk0LTg4ODYtNGE1MDE5OTA1OTdmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMxVDE2Mzk1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI5OWE4MGYzMzQ2MzE5ZTk2MGU2M2YxNmIyMTBjNGM2ZWYwZWZiOWQzYjFlZjBjOWU1MjkyMDA1MmVhZTRmZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.b9RZNlbiBcZ4Srist0_VKk2fY702JbgGu_tYklGNebI) ![image](https://private-user-images.githubusercontent.com/55346310/381803173-aaba088a-fa5c-4ce9-942a-e2d5335ffbe6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzOTMwOTMsIm5iZiI6MTczMDM5Mjc5MywicGF0aCI6Ii81NTM0NjMxMC8zODE4MDMxNzMtYWFiYTA4OGEtZmE1Yy00Y2U5LTk0MmEtZTJkNTMzNWZmYmU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMxVDE2Mzk1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1MmFmMmE1MGU0YjMxZTE1ZTJjODE5OWUxMzRmNjYyNTAzNjZkOWViNWU2NzZhZDJiMmQ2ZDM0OTQwMGE3M2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GNXm7SIeJoiMwBGa17dYr7fqFx9T2I0gGigDpmcC7Vs)

Its in the first screenshot?

Reply:
Is zombie not different to the redstone lamps?